src/frontend/Player/createPlyrPlayer.ts

```diff
@@ -79,6 +112,10 @@ export const createPlyrPlayer = (
     settings,
   });
 
+  if (sources) {
```

Comment:
Maybe a comment here to remind the reader sources are only defined when there is no ABR